src/components/footer.js

```diff
@@ -1,74 +1,97 @@
 // External Dependencies
+import PropTypes from 'prop-types';
 import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
 
 // Internal Dependencies
 import FacebookSvg from './shared/icons/facebook.svg';
 import SlackSvg from './shared/icons/slack.svg';
 import TwitterSvg from './shared/icons/twitter.svg';
 
 // Local Variables
-const iconContainerStyles = {
-  margin: '0 16px',
+const propTypes = {
+  classes: PropTypes.shape({
+    copyright: PropTypes.string.isRequired,
+    footer: PropTypes.string.isRequired,
+    footerContent: PropTypes.string.isRequired,
+    icon: PropTypes.string.isRequired,
+    iconContainer: PropTypes.string.isRequired,
+    link: PropTypes.string.isRequired,
+  }).isRequired,
 };
 
-const iconStyles = {
-  height: 32,
-  width: 32,
-};
+const styles = theme => ({
+  copyright: {
+    color: theme.palette.grey[300],
+    fontSize: 15,
+  },
+  footer: {
+    background: '#20232A',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: '1rem',
+  },
+  footerContent: {
+    display: 'flex',
+    fontFamily: 'Roboto',
+    justifyContent: 'center',
+    margin: '0 auto',
+    maxWidth: 960,
+    padding: '0.5rem',
+  },
+  icon: {
+    height: 32,
+    width: 32,
+  },
+  iconContainer: {
+    margin: '0 16px',
```

Comment:
curious do you have any opinions on the array format vs. string for these shorthand properties?

Reply:
Not sure that I have heard it described that way before. Can you give more detail about the difference?

Reply:
Oh I'm talking about this https://cssinjs.org/jss-plugin-expand?v=v10.0.0-alpha.13#using-arrays-for-space-separated-values

Curious if y'all had a convention and rationale for picking one or the other

Reply:
IIRC it wasn't part of core until recently so you may have just never used the plugin before :)

Reply:
(the docs seem to disappear if you pick v9.8)

Reply:
Oh, I haven't seen that. I know that Material-UI latest is using an older version of react-jss under the hood, so we wouldn't have had access to it. Pretty interesting!